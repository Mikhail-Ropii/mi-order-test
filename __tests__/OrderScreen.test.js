import React from "react";
import renderer from "react-test-renderer";
// import OrderScreen from "../src/screens/OrderScreen";
import { FlatList } from "react-native";

// it("render FlatList", () => {
//   const tree = renderer.create(<FlatList data={["item",]} />).toJSON();
//   expect(tree).toMatchSnapshot();
// });
