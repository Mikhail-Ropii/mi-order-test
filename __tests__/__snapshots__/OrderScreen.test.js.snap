// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render FlatList 1`] = `
<RCTScrollView
  data={
    [
      "item",
    ]
  }
  getItem={[Function]}
  getItemCount={[Function]}
  keyExtractor={[Function]}
  onContentSizeChange={[Function]}
  onLayout={[Function]}
  onMomentumScrollBegin={[Function]}
  onMomentumScrollEnd={[Function]}
  onScroll={[Function]}
  onScrollBeginDrag={[Function]}
  onScrollEndDrag={[Function]}
  removeClippedSubviews={false}
  renderItem={[Function]}
  scrollEventThrottle={50}
  stickyHeaderIndices={[]}
  viewabilityConfigCallbackPairs={[]}
>
  <View>
    <View
      onLayout={[Function]}
      style={null}
    />
  </View>
</RCTScrollView>
`;
